### ISSUE TITLE
Failed to proceed if domain contains '-'

### ISSUE THREAD

**Issue by evianzhow** [fenying/le-alidns#2]
https://github.com/fenying/le-alidns/blob/5233788dd9e11e5444d7cd544493b89999cba2b8/actions/create-dns-record.sh#L21

```
CERTBOT_DOMAIN="www.baidu.com"
API_DomainName=$(echo $CERTBOT_DOMAIN | grep -P "\w+\.\w+$" -o)
DomainRecord=$(echo $CERTBOT_DOMAIN | grep -P ".+(?=\.\w+\.\w+$)" -o)
echo $API_DomainName
>> baidu.com
echo $DomainRecord
>> git
```

My domain contains '-' and the script failed to process it. 

```
CERTBOT_DOMAIN="www.bai-du.com"
API_DomainName=$(echo $CERTBOT_DOMAIN | grep -P "\w+\.\w+$" -o)
DomainRecord=$(echo $CERTBOT_DOMAIN | grep -P ".+(?=\.\w+\.\w+$)" -o)
echo $API_DomainName
>> du.com
echo $DomainRecord
>> 
```

**Comment by evianzhow** [comment]
Suggestion:

```
API_DomainName=$(echo $CERTBOT_DOMAIN | grep -P "[a-z0-9|-]+\.[a-z]+$" -o)
DomainRecord=$(echo $CERTBOT_DOMAIN | grep -P "([a-z0-9|-]+)*(?=\.[a-z0-9|-]+\.[a-z]+)" -o)
```

**Comment by fenying** [comment]
Sorry for my delay...
Yes, I did miss the dash charactor... I will fixed it later. 0_0
Actually you can submit a PR if you have a solution for this problem, and I'll be glad to accept it. ^_^

**Comment by fenying** [comment]
This problem has been fixed in latest version v0.2.3, thanks a lot.
